applications/GeoMechanicsApplication/tests/test_strain_measures/diff_order_elements/column_small_deformation_logarithmic_strain/ProjectParameters.json

```diff
@@ -0,0 +1,159 @@
+{
+    "problem_data": {
+        "problem_name":         "column_small_displacement_logarithmic_strain",
+        "start_time":           0.0,
+        "end_time":             1.0,
+        "echo_level":           1,
+        "parallel_type":        "OpenMP",
+        "number_of_threads":    1
+    },
+    "solver_settings": {
+        "solver_type":                        "U_Pw",
+        "model_part_name":                    "PorousDomain",
+        "domain_size":                        2,
+        "model_import_settings":              {
+            "input_type":       "mdpa",
+            "input_filename":   "column_small_displacement_logarithmic_strain"
+        },
+        "material_import_settings":              {
+            "materials_filename":       "MaterialParameters.json"
+        },
+        "time_stepping":              {
+            "time_step":                1.0,
+            "max_delta_time_factor":    1000
+        },
+        "buffer_size":                        2,
+        "echo_level":                         1,
+        "clear_storage":                      false,
+        "compute_reactions":                  false,
+        "move_mesh_flag":                     true,
+        "reform_dofs_at_each_step":           false,
+        "nodal_smoothing":                    false,
+        "block_builder":                      true,
+        "solution_type":                      "Quasi-Static",
+        "scheme_type":                        "Newmark",
+        "reset_displacements":                true,
+        "newmark_beta":                       0.25,
+        "newmark_gamma":                      0.5,
+        "newmark_theta":                      0.5,
+        "rayleigh_m":                         0.0,
+        "rayleigh_k":                         0.0,
+        "strategy_type":                      "newton_raphson",
+        "convergence_criterion":              "displacement_criterion",
+        "displacement_relative_tolerance":    1.0E-4,
+        "displacement_absolute_tolerance":    1.0E-9,
+        "residual_relative_tolerance":        1.0E-4,
+        "residual_absolute_tolerance":        1.0E-9,
+        "water_pressure_relative_tolerance":  1.0E-4,
+        "water_pressure_absolute_tolerance":  1.0E-9,
+        "min_iterations":                     6,
+        "max_iterations":                     15,
+        "number_cycles":                      1,
```

Comment:
The changes to number_cycles, reduction_factor and increase_factor are for the following reasons:
1 bringing number_cycles down reduced waiting time when I was not doing correct things yet. Otherwise I would have waited for more cycles not bringing a solution. Now that I have correct and converging results, those happen in 1 cycle, so more are not necessary.
2 that changes to the reduction_factor and increase_factor are such that the time stepping scheme just does equal size steps.